game/dialogs/script-greetings.rpy

```diff
@@ -287,7 +287,7 @@ label s_return_long_absence:
     $ persistent._fae_long_absence = False
 
     show desknote
-    call fae_showpoem(long_wait) from _call_fae_showpoem
+    call showpoem(long_wait) from _call_fae_showpoem
```

Comment:
Did the showmadechoice function not work here for some reason?

Reply:
Yes.